src/sgame/sg_buildable.cpp

```diff
@@ -35,6 +35,11 @@ static Cvar::Cvar<bool> g_indestructibleBuildables(
 		"g_indestructibleBuildables",
 		"make buildables impossible to destroy", Cvar::NONE, false);
 
+static Cvar::Range<Cvar::Cvar<float>> g_buildPointMinerQueueFraction(
```

Comment:
I can't help but think this name is very, very long. Would be nice to have shorter names, as I think  long names hurt readability (but ofc, I'd rather have long names than "a", "b", "c"...).

I understand that all BP-related cvars do use that kind of names, but I think it could be nice to do a PR to reduce "buildPoint" to "BP" after this is merged. BP is also the term everyone (players and devs) use, so I do not think it would be a problem, and I believe it would help mitigate the readability issue.

Reply:
Actually, it seems we have a double standard on that point: there are cvars using the BP abbreviation:

* g_BPInitialBudget
* g_BPBudgetPerMiner
* g_BPRecoveryInitialRate
* g_BPRecoveryRateHalfLife